### PR TITLE
Added samyakbvs photo and bio

### PR DESCRIPTION
**Pull request by samyakbvs**

### CODE REVIEW

**Comment by isuruAb** [comment]
@samyakbvs  could you please add a square shaped photo

**Comment by samyakbvs** [comment]
screen_shot_2017-12-06_at_3_02_07_pm

**Comment by samyakbvs** [comment]
I don't know for some reason another photo is not getting uploaded

**Comment by samyakbvs** [comment]
I hope now its fine?

**Comment by anshumanv** [comment]
The image is perfect, please replace it with the initially uploaded image.

**Comment by samyakbvs** [comment]
The previous one is not getting delted

**Comment by anshumanv** [comment]
Add a commit to replace the older with the newly uploaded image, it will work, please explore. :smile:

**Review by isuruAb — COMMENTED** [review]
please replace the previous image with new image and  squash your commits.
follow this tutorial to squash your commits
https://www.youtube.com/watch?v=vp7-NeURmFg&t=35s

**Comment by samyakbvs** [comment]
Now shall I make the code changes?

**Comment by anshumanv** [comment]
You need to delete one image, the one you uploaded before the recent one.

**Comment by samyakbvs** [comment]
i deleted it

**Comment by anshumanv** [comment]
It's not deleted yet, please see - https://github.com/fossasia/gci17.fossasia.org/pull/140/files
3 files.

**Comment by samyakbvs** [comment]
Is everything alright now?

**Comment by anshumanv** [comment]
Yes @samyakbvs, thanks for learning along the way.

**Comment by samyakbvs** [comment]
Now i have to edit the code?

File: _data/students.yml [hunk 1]
```diff
@@ -88,3 +88,9 @@
   github: aryankedare
   image: aryankedare.jpg
   twitter: KedareAryan
+  
+  - name: Samyak Jain
```
Comment (anshumanv) [inline _data/students.yml]: Please remove the space before `-`

**Comment by anshumanv** [comment]
No, you don't need to edit the code, you just needed to specify details in the student's file(which you did), the templating engine will automatically take your details and display on the website.

**Comment by samyakbvs** [comment]
Oh so I am done?

**Comment by anshumanv** [comment]
Almost, just follow the review I made, make a minor change and this can be merged! :smile:

**Review by anshumanv — APPROVED** [review]
Awesome!

**Comment by anshumanv** [comment]
Thanks so much @samyakbvs, for keeping a learning attitude and incorporating changes that were requested! :smile: 

Keep it up!

**Comment by anshumanv** [comment]
You can see the commit tab with several commits, try to squash your commits from next time as it's the best practice to be followed! :smile:

**Comment by samyakbvs** [comment]
😄

**Pull request merged**